spec/features/recipes/welcome_page_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
```

Comment:
Please DRY this up. We can have `.rspec` (read the docs) file in the root with this configuration.